tests/unittest/tools/test_config_database_sync.py

```
@@ -0,0 +1,126 @@
+import os
+import sys
+import tempfile
+import unittest
+from pathlib import Path
+
+# Add scripts directory to path
+REPO_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../.."))
+SCRIPTS_DIR = os.path.join(REPO_ROOT, "scripts")
+sys.path.insert(0, SCRIPTS_DIR)
+
+from generate_config_database_tests import (  # noqa: E402
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)